src/components/ActivitiesList/ActivitiesList.tsx

```diff
@@ -15,27 +17,38 @@ interface ActivitiesListProps {
 }
 
 export default function Activities(props: ActivitiesListProps): ReactElement {
+  const [activitiesList, setActivitiesList] = useState(props);
+
   return (
-    <div className={styles.list__main}>
-      <h1 className={styles.list__h1}>Your Activities</h1>
-      <ul>
-        {props.activities.map((activity) => (
-          <li key={activity.id} className={styles.list__card}>
-            <h2>
-              <strong>Activity:</strong>: {activity.name}
-            </h2>
-            <p>
-              <strong>Description:</strong>: {activity.description}
-            </p>
-            <p>
-              <strong>Category:</strong>: {activity.category}
-            </p>
-            <p>
-              <strong>Vote Count:</strong>: {activity.voteCount}
-            </p>
-          </li>
-        ))}
-      </ul>
-    </div>
+    <ul>
+      {props.activities.map((activity) => (
+        <li key={activity.id} className={styles.list__card}>
+          <h2>Activity: {activity.name}</h2>
+          <p>Description: {activity.description}</p>
+          <p>Category: {activity.category}</p>
+          <p>Place: {activity.location}</p>
+          <p>When: {activity.date}</p>
+          <p>
+            Time: {activity.timeStart} - {activity.timeEnd}
+          </p>
```

Comment:
Activities aren't actually going to have Dates, Times, or Locations - Those will be part of the Event that the Activities are eventually part of. Please delete from the render as well as the mock data